content/en/news/releases/1.7.x/announcing-1.7/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Announcing Istio 1.7
+linktitle: 1.7
+subtitle: Major Update
+description: Istio 1.7 release announcement.
+publishdate: 2020-08-14
+release: 1.7.0
+skip_list: true
+aliases:
+    - /news/announcing-1.7.0
+    - /news/announcing-1.7
+---
+
+We are pleased to announce the release of Istio 1.7!
+
+{{< relnote >}}
+
+With this release, we continue the path we charted earlier this year in
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and we have added other goodies as
+well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Item 1
+
```

Comment:
missing content?

Reply:
yeah, this will be written by @oaktowner .